stage0/README.md

```diff
@@ -37,3 +37,39 @@ To use the binary, pass it to `qemu -bios`; for example:
 ```shell
 qemu-system-x86_64 -nodefaults -nographic -no-reboot -machine microvm -bios stage0.bin
 ```
+
+Memory layout
+
+Stage 0 only maps the first 1GiB of physical RAM using an identity mapping. The
+Stage 0 ROM image is mapped just below the 4GiB boundary.
+
+The diagram below assumes the guest VM has at least 3GiB of guest-physical RAM
```

Comment:
Can you give an indication where / how the assumption of 3 GiB is relevant in the diagram?

Reply:
The assumption is just so that there does not have to be another line below 3GiB where the physical memory ends